TaskManager/src/Kreta/TaskManager/Infrastructure/Symfony/Command/ProjectFixturesCommand.php

```diff
@@ -61,7 +68,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     Uuid::generate()
                 )
             );
+            $i++;
+            $progress->advance();
         }
-        $output->writeln('Project population is successfully done');
+        $progress->finish();
+        $output->writeln('');
```

Comment:
Yes, progress bar does not jump to a new line when it finishes otherwise, didnt found a better solution.